data/mods/survivorweaps/classes.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "abstract": "rifle_pipe",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "pipe rifle",
+    "clip_size": 1,
+    "valid_mod_locations": [ [ "accessories", 2 ], [ "barrel", 1 ], [ "muzzle", 1 ], [ "stock", 1 ] ],
+    "proportional": { "reload": 4.0 },
+    "extend": { "flags": [ "RELOAD_ONE", "RELOAD_EJECT" ] }
+  }
```

Comment:
> Doesn't set symbol or color, making things look like white '#'s.

As of ca94f1a set by the base classes

> It could be worth adding a warning on finalize if symbol and color aren't set.

A check for undefined symbol is included in 11aaa39 which helps identify faults such as the one corrected for `fish_cooked`.